### ISSUE TITLE
Some links are distinguished from surrounding text only by color

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks#1638]
### Description

At several places in the Pressbooks authoring interface, color is the only differentiating indicator between link text and regular text. This makes links difficult to use for those with color-blindness or users who cannot perceive color.

We should add a secondary indicator such as underlining, italics, etc. This can be done via CSS. Or, if only color is used, the contrast between link and regular text should be at least 3:1 (currently the contrast ratio between links and surrounding text is 1.5:1) and an additional differentiation (e.g. underline) should be provided on hover and on focus. 

A few places where this has been observed: the Users section on the Dashboard and the title of individual posts in the `Pressbooks News` widget:
![Screen Shot 2019-04-03 at 12 18 12 PM](https://user-images.githubusercontent.com/13485451/55499056-a9351200-560a-11e9-9994-0db8ddf32023.png)
![Screen Shot 2019-04-03 at 12 18 17 PM](https://user-images.githubusercontent.com/13485451/55499055-a89c7b80-560a-11e9-832d-82c225668f4b.png)

**Comment by SteelWagstaff** [comment]
Should be addressed by https://github.com/pressbooks/pressbooks/pull/1681. @dannylonglegs can you confirm?
